cross/openzwave/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = openzwave
+PKG_VERS = $(PKG_SVN_REV)
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = svn
+PKG_DOWNLOAD_VCS_CLEAN = nope
```

Comment:
I was mostly following the guidelines of the tutorial on Domoticz.com about how to get OpenZWave working with with Domoticz on Syno, which specifically used the SVN version. But I'm happy to test with the stable version to see if it still works.